### ISSUE TITLE
Re-entering crop doesn't set flip correctly

### ISSUE THREAD

**Issue by xinatanil** [guoyingtao/Mantis#340]
1. Edit image by simply h-flipping it
2. Save transformation and crop info using delegate methods
3. Open the same image with the provided transformation and crop
4. Everything works as expected, but the image is not flipped.

```
var config = Mantis.Config()
config.cropToolbarConfig.mode = .embedded
config.showAttachedCropToolbar = false
config.cropViewConfig.cropMaskVisualEffectType = .dark

if let transformation = self.cropData?.transformation {
    config.cropViewConfig.presetTransformationType = .presetInfo(info: transformation)
}
self.cropController = Mantis.cropViewController(image: image, config: config)
self.cropController?.delegate = self
```

**Comment by guoyingtao** [comment]
@xinatanil 
Thanks for pointing it out. Please check PR #342 to see if it solved the issue.

**Comment by guoyingtao** [comment]
It is fixed in Mantis 2.15.0.
